### ISSUE TITLE
Image for source / raw / derivatives

### ISSUE THREAD

**Issue by Remi-Gau** [bids-standard/bids-starter-kit#105]
I had made this figure to add it to the BIDS specs. But before I make a pull request there I was thinking that we could work on it / add it to the starter kit and then make a PR to the specs.

What do you think?

[link on google drive](https://drive.google.com/open?id=19fPIg-QHgfFeAoENFYj5BxPyK78TI8Sv)

[SVG file](https://drive.google.com/open?id=18LeXnzn8Mfwb8nLg-M3huAEAbesnnuO6) made with [inkscape](https://inkscape.org/) if you want to edit it further. 

BTW: Is there a good way to version control figure making? git-annex?

![bids_rawderivativesource](https://user-images.githubusercontent.com/6961185/47993749-9a484300-e0f0-11e8-8385-924e06b97e96.png)

**Comment by dorahermes** [comment]
Very nice!

**Comment by KirstieJane** [comment]
Yep! Ditto! In fact - I'm about to use it in a talk at SFN in 1 hours time! GREAT TIMING @Remi-Gau 🙌

**Comment by Remi-Gau** [comment]
Awesome! Glad it is being used!
I think @chrisfilo had mentioned (in the google doc days of the BIDS spec) that there should be less detail in the derivatives so I could make something like that for the main specs!

**Comment by emdupre** [comment]
Re: version control for figures, I'd agree that using git-annex is a good option ! I've found this walk-through helpful in the past for [setting up special remotes](https://git-annex.branchable.com/special_remotes/).

Could you clarify what you mean by "less detail in the derivatives" ? Just not sure how it relates to making something different for the main spec, but I'm excited to see this move forward either way ! :rocket: Thanks for bringing this up !

**Comment by dorahermes** [comment]
Thank you!
an image dir in root and BIDS folder hierarchy sounds like good places to me

**Comment by Remi-Gau** [comment]
BTW: added the image to the [wiki ](https://github.com/bids-standard/bids-starter-kit/wiki/The-BIDS-folder-hierarchy#bids-source-rawdata-derivatives).
Looks a bit huge ATM (should it be resized or is there the way to do this on the fly in markdown?) and also the link to the image will have to be changed once #106 has been merged.
